iceddev/git-splainin
# Chrome error handling

#### What's this PR do?

Handles chrome errors on the options page, creates a module to convert chrome callback to node-style callbacks. Alphabetizes the handle functions inside of template-tab for my own sanity. Finally extracts error message finding logic
#### Where should the reviewer start?

Start with src/lib/chromeBack.js to understand how we're converting chrome callbacks to node style. Then move into the template tab.  
#### How should this be manually tested?

After loading the extension go to the options page and load www.google.com then try and save it. You should receive an error about maximum bytes or quota being exceeded 
#### Any background context you want to provide?

JSCS is failing due to: https://github.com/jscs-dev/node-jscs/issues/862 Eslint uses Espree which has this issue fixed.

Background chrome errors still aren't handled or displayed to the user, could use some guidance on best practices with this, recursively calling the api's on failure seemed heavy-handed.
#### What are the relevant tickets?

closes #23
